src/Sensor.cpp

```diff
@@ -114,8 +114,7 @@ void Barometer::update_data(double tStep) {
         new_data_ = true;
 
         if (inject_noise_) {
-            noise_ = normal_dist_(generator_);
-            data_ += noise_;
+            data_ = normal_dist_(generator_);
```

Comment:
Hmm. Why did you get ride of the `noise_` variable for the Barometer? I see it was deleted from `Sensor.h`.